depCheck.go

```diff
@@ -9,7 +9,7 @@ import (
 	alpm "github.com/jguer/go-alpm"
 )
 
-func (dp *depPool) checkInnerConflict(name string, conflict string, conflicts mapStringSet) {
```

Comment:
I'm not sure what's more idiomatic for go but my personal opinion is to only skip the return types if all returns are the same type.

Reply:
OK. Combined parameters in some functions are rejected in 14ac7562981a20bbc0a91c7fcdafc7a1c6b6ab9c.